libraries/AP_ExternalAHRS/AP_ExternalAHRS.h

```diff
@@ -206,6 +213,8 @@ class AP_ExternalAHRS {
 
     // true when user has disabled the GNSS
     bool gnss_is_disabled;
+
+    ins_data_message_t pkt;
```

Comment:
This shouldn't be out here.

Have the backend fill in the new field in its own packet, default it to not temperature calibrated.